keyboards/gergo/keymaps/default/rules.mk

```diff
@@ -0,0 +1,57 @@
+#----------------------------------------------------------------------------
+# make gergo:germ:dfu
+# Make sure you have dfu-programmer installed!
+#----------------------------------------------------------------------------
+
+# Source includes
```

Comment:
Lines 6-15 aren't needed at the keymap level; neither are Lines 39-57.

`keyboards/gergo/rules.mk` gets processed first at compile time, then `keyboards/gergo/keymap/<keymapname>/rules.mk` can override settings if needed (like if someone buys a keyboard PCB that has solder pads for an RGB underglow strip, but doesn't come with the RGB LEDs).

If you have rules that are required to be set a certain way for the keyboard to work, they should be at the keyboard level but should *not* be in this file. (`CUSTOM_MATRIX` rule should probably go too for that reason.)